File: eAirFlow.Model/Models/Airplane.cs
using System;
using System.Collections.Generic;

namespace eAirFlow.Model.Models
{

    public class Airplane
    {
        public int AirplaneId { get; set; }

        public string? Model { get; set; }

        public int? TotalSeats { get; set; }
        public int? AirlineId { get; set; }

    }
}

File: UI/eairflow_desktop/lib/models/airplane.dart
class Airplane {
  int? airplaneId;
  String? model;
  int? totalSeats;
  int? airlineId;

  Airplane({
    this.airplaneId,
    this.model,
    this.totalSeats,
    this.airlineId,
  });

  factory Airplane.fromJson(Map<String, dynamic> json) => Airplane(
        airplaneId: json['airplaneId'] as int?,
        model: json['model'] as String?,
        totalSeats: json['totalSeats'] as int?,
        airlineId: json['airlineId'] as int?,
      );

  Map<String, dynamic> toJson() => {
        'airplaneId': airplaneId,
        'model': model,
        'totalSeats': totalSeats,
        'airlineId': airlineId,
      };
}